ONSdigital/response-operations-ui
# Fixed Sand & Gravel 500 error pages

# Motivation and Context
When a internal user clicks on messages for the Sand & Gravel survey, some of the links don't work because the survey gets sent through with spaces. The aim of the pr is to fix this and remove the whitespace so the user can click on Sand & Gravel messages. While looking at Sand & Gravel in response ops, I also found some other links that don't work because of the same bug so they've been fixed as well.
# What has changed
- Added a method to remove whitespace for a survey

# How to test?
- Execute a collection exercise for Sand & Gravel, make sure to click most of the options like attempting to add a reminder, change information for the collex and clicking on cancel/back when trying to add a sample/CI.
- Create a conversation for Sand & Gravel and make sure to click on open and closed messages for that survey, attempt to close a conversation and click back buttons on the conversation view to make sure it goes back correctly.
- Run with ATs
# Links
[Trello](https://trello.com/c/JBFvWCIM)
